README.md

```diff
@@ -2,34 +2,142 @@
 
 ## Getting started
 
-Fork and clone
+Fork and then clone
+
+Navigate into project
 
 ```shell
 cd bobabot
 ```
+Initialize your python virtual environment
 
-Navigate into project
+_NOTE_: This is only necessary for new environment setups_
 
 ```shell
 python -m venv venv
 ```
 
-Initialize your python virtual environment
-
-_note: This is only necessary for new environment setups_
+Activate the python virtual environment
 
 ```shell
 source venv/bin/activate
 ```
 
-Activate the python virtual environment
+Install project dependencies into virtual environment
 
 ```shell
 pip install -r requirements.txt
 pip install -r requirements.dev.txt
 ```
 
-Install project dependencies into virtual environment
+### Creating a discord bot & server
+
+First, create a new server
+
+Enter server settings
+
+Under Community, register your server as a community server
+
+Add a new forum text channel, make the name match your .env variable *DISCORD_CHANNEL_NAME*
+
+Under the settings for this new channel, add a couple tags. These represent different fields/roles.
+
+```shell
+# Example Tags
+SWE, PM, DS, UI/UX
+```
+
+Additionally, register for a discord developer account and create a new bot application.
+
+Invite your bot to your server with the following set of permissions:
+
+<div style="padding-bottom: 20px;">
+    <li> Manage Server
+    <li> Manage Channels
+    <li> Send Messages
+    <li> Create Public Threads
+    <li> Manage Messages
+    <li> Embed Links
+    <li> Use Slash Commands
+</div>
```

Comment:
![image](https://github.com/BobaTalks/bobabot/assets/116035879/1b41344e-1de0-4156-9f20-ad5b87e4f9d1)

![Capture](https://github.com/BobaTalks/bobabot/assets/116035879/991a2679-ecd0-4239-8966-8e0831c72d40)

I think this doesn't format entirely correctly in markdown since its in a div. Will keep as  < li> for now, but can swap for list in markdown as well.

Reply:
yah not entirely sure what the purpose of a div is here